Tests/macOS/FamilyViewControllerSharedTests.swift

```diff
@@ -0,0 +1,99 @@
+import XCTest
+
+@testable import Family_macOS
+@testable import Family_Shared
+
+class FamilyViewControllerSharedTests: XCTestCase {
+    func testMarginsAndPadding() {
+        let childViewController = ChildViewController()
+        childViewController.view.frame.size = CGSize(width: 500, height: 500)
+        let insets = Insets(top: 10, left: 10, bottom: 10, right: 10)
+        let backgroundView = View()
+        let container = ContainerViewController(builder: { instance in
+            instance.body {
+                instance.add(childViewController)
+                    .margin(insets)
+                    .padding(insets)
+                    .background(.view(backgroundView))
+            }
+        })
+        container.view.frame.size = CGSize(width: 500, height: 1000)
+        container.scrollView.frame.size = CGSize(width: 500, height: 1000)
+        #if os(macOS)
+        let window = NSWindow(contentViewController: container)
+        container.scrollView.layoutViews(withDuration: nil, force: true, completion: nil)
+        window.makeKeyAndOrderFront(nil)
+        #else
+        container.loadViewIfNeeded()
+        container.scrollView.layoutViews()
+        #endif
+
+
+        let margins = container.margins(for: childViewController.view)
+        let padding = container.padding(for: childViewController.view)
+
+        XCTAssertEqual(margins.top, insets.top)
+        XCTAssertEqual(margins.left, insets.left)
+        XCTAssertEqual(margins.right, insets.right)
+        XCTAssertEqual(margins.bottom, insets.bottom)
+
+        XCTAssertEqual(padding.top, insets.top)
+        XCTAssertEqual(padding.left, insets.left)
+        XCTAssertEqual(padding.right, insets.right)
+        XCTAssertEqual(padding.bottom, insets.bottom)
+
+        XCTAssertEqual(container.scrollView.bounds, CGRect(origin: .zero, size: CGSize(width: 500, height: 1000)))
```

Comment:
I removed this line on `master`, if you do the same I think your branch should work.